utils/frame/remote-externalities/Cargo.toml

```diff
@@ -14,6 +14,7 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 jsonrpsee-http-client = { version = "=0.2.0-alpha.6", default-features = false, features = ["tokio02"] }
+jsonrpsee-ws-client = { version = "=0.2.0-alpha.6", default-features = false }
```

Comment:
the ws-client doesn't have any features, it spawns the background task by itself.